doc/releases/changelog-dev.md

```diff
@@ -259,7 +259,8 @@
 
 <h3>Bug fixes</h3>
 
-* `qml.CSWAP` and `qml.CRot` now define `control_wires`, and `qml.SWAP`
```

Comment:
This line was from a merge remnant and the actual full entry is below, so I tidied it up.